README.md

```diff
@@ -131,8 +131,10 @@ the following libraries.
   - [Conversation with ReAct Agent](./examples/conversation_with_react_agent)
   - [Conversation in Natural Language to Query SQL](./examples/conversation_nl2sql/)
   - [Conversation with RAG Agent](./examples/conversation_with_RAG_agents)
+  - [Conversation with SWE Agnet](./examples/swe_agent/)
```

Comment:
Add this figure <img src="https://img.alicdn.com/imgextra/i3/O1CN01SFL0Gu26nrQBFKXFR_!!6000000007707-2-tps-500-500.png" alt="new" width="30" height="30"/>